src/Tribe/REST/Endpoints/DELETE_Endpoint_Interface.php

```diff
@@ -0,0 +1,26 @@
+<?php
+
+interface Tribe__REST__Endpoints__DELETE_Endpoint_Interface {
+	/**
+	 * Handles DELETE requests on the endpoint.
+	 *
+	 * @param WP_REST_Request $request
+	 *
+	 * @return WP_Error|WP_REST_Response An array containing the data of the trashed post on
+	 *                                   success or a WP_Error instance on failure.
+	 */
+	public function delete( WP_REST_Request $request );
+
+	/**
+	 * Returns the content of the `args` array that should be used to register the endpoint
+	 * with the `register_rest_route` function.
+	 *
+	 * @return array
+	 */
+	public function DELETE_args();
+
+	/**
+	 * @return bool Whether the current user can delete or not.
+	 */
+	public function can_delete(  );
```

Comment:
Matt mentioned removing the extra space in the function method `(  )` >> `()` in the other PR, can you do that here too?